src/SOS/lldbplugin/services.cpp

```diff
@@ -803,21 +849,20 @@ LLDBServices::ReadVirtual(
                     lldb::SBData sectionData = section.GetSectionData(offset - loadAddr, bufferSize);
                     if (sectionData.IsValid())
                     {
-                        read = sectionData.ReadRawData(error, 0, buffer, bufferSize);
-                        found = true;
-                        break;
+                        bytesRead += sectionData.ReadRawData(error, 0, buffer, bufferSize);
```

Comment:
Any reason to prefer goto over break?

Reply:
It gets rid of the "found" flag and we already goto exit everywhere else (and on success now).